corehq/messaging/scheduling/models/content.py

```diff
@@ -82,26 +46,71 @@ def send(self, recipient, schedule_instance, logged_event, case=None):
             return
 
         message = self.get_translation_from_message_dict(
+            logged_event.domain,
             self.message,
-            schedule_instance.memoized_schedule,
             recipient.get_language_code()
         )
-        message = self.render_message(message, logged_subevent, case=case)
+        message = self.render_message(message, recipient, logged_subevent)
 
-        send_sms_for_schedule_instance(schedule_instance, recipient, phone_number, message, logged_subevent)
+        self.send_sms_message(logged_event.domain, recipient, phone_number, message, logged_subevent)
         logged_subevent.completed()
 
 
 class EmailContent(Content):
     subject = old_jsonfield.JSONField(default=dict)
     message = old_jsonfield.JSONField(default=dict)
 
-    def send(self, recipient, schedule_instance, logged_event, case=None):
-        print('*******************************')
-        print('To:', recipient)
-        print('Subject: ', self.subject)
-        print('Message: ', self.message)
-        print('*******************************')
+    TRIAL_MAX_EMAILS = 50
+
+    def render_subject_and_message(self, subject, message, recipient):
+        renderer = self.get_template_renderer(recipient)
+        return renderer.render(subject), renderer.render(message)
+
+    def send(self, recipient, logged_event):
+        email_usage = EmailUsage.get_or_create_usage_record(logged_event.domain)
+        is_trial = domain_is_on_trial(logged_event.domain)
+
+        logged_subevent = logged_event.create_subevent_from_contact_and_content(
+            recipient,
+            self,
+            case_id=self.case.case_id if self.case else None,
+        )
+
+        subject = self.get_translation_from_message_dict(
+            logged_event.domain,
+            self.subject,
+            recipient.get_language_code()
+        )
+
+        message = self.get_translation_from_message_dict(
+            logged_event.domain,
+            self.message,
+            recipient.get_language_code()
+        )
+
+        try:
+            subject, message = self.render_subject_and_message(subject, message, recipient)
+        except:
+            logged_subevent.error(MessagingEvent.ERROR_CANNOT_RENDER_MESSAGE)
+            return
+
+        subject = subject or '(No Subject)'
```

Comment:
i had thought about this, and unfortunately it's not easily translated in a way that's consistent with the rest of the reminder translations. for now i chose to leave it as the old framework was behaving and just not translate.

(we also shouldn't get to this point since subject is required from the ui, but it might happen in a rare situation if the subject only references a variable like {case.property} which renders to empty string at run time)